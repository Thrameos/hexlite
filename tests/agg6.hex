d(1). d(2).
p(1,2). p(1,3).
p(2,3). p(2,4).
res(X, S) :- S = #sum{ Y : p(X,Y) }, d(X).

q(2,1). q(3,1).
q(3,2). q(4,2).
res2(X, S) :- S = #sum{ Y : q(Y,X) }, d(X).
