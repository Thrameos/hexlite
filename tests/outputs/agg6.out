{res(1,5),res(2,7),res2(1,5),res2(2,7)}
